packages/universal-components/src/RangeDatePicker/RenderDay.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { format, isSameDay, isBefore } from 'date-fns';
+
+import { StyleSheet } from '../PlatformStyleSheet';
+import { designTokens } from '../DesignTokens';
+import { Text } from '../Text';
+import { Touchable } from '../Touchable';
+import RangeDateConfig from './RangeDateConfig';
+
+type Props = {|
+  +day: Date,
+  +onPress: Date => void,
+  +price?: ?string,
+  +selectedDate: Date,
+|};
+
+const DayPrice = ({ price }) => (
+  <View>
+    <Text numberOfLines={1} style={styles.price}>
+      {price}
+    </Text>
+  </View>
+);
+
+export default function RenderDay({
+  day,
+  onPress,
+  price,
+  selectedDate,
+}: Props) {
+  function handlePress() {
+    onPress(day);
+  }
+
+  const isDaySelected = isSameDay(selectedDate, day);
+  const isDayInPast = isBefore(day, new Date());
+
+  return (
+    <View style={{ flex: 1 }}>
+      <Touchable onPress={handlePress} disabled={isDayInPast}>
```

Comment:
Disabled condition should be extended, past days are ok, but empty day slots (shown below) are still clickable. Best way to try it is in web version.

<img width="386" alt="Screenshot 2019-03-27 at 10 47 12" src="https://user-images.githubusercontent.com/2660330/55066290-c5bad280-507d-11e9-9e24-90cb0c9c0e5f.png">

Reply:
Nice bug 🐛 now it should be fixed

Reply:
Thanks, it's ok now.